docs/31-long-running-job.md

```diff
@@ -81,9 +83,14 @@ Once job is completed successfully, you can call getJobResult to retrieve the jo
 
 ### Clean up
 
-Once you get the job result, you can delete the job.
+Once you get the job result, you can delete the job and its result.
+```R
+  deleteJob(jobId)
```

Comment:
doAzureParallel::deleteJob(jobId) to differentiate that it's not the rAzureBatch::deleteJob(jobId)

Might want to add docs about the difference

Reply:
My understanding is that we want to discourage customers to use rAzureBatch::deleteJob, if they use it, it can cause problem to our current design of treating job & result as an atomic entity. We can discuss this later.

Reply:
added namespaces to all api calls.